articles/tales-from-fleet-security-securing-bank-accounts-from-business-email-compromise.md

```diff
@@ -0,0 +1,112 @@
+
+## Email configuration
+While good email hygiene can't stop BEC, it can at least reduce the number of fraudulent emails that make it through and force criminals to work harder.
+
+1. Require secure two-factor authentication (2FA) to access email, such as [FIDO U2F](https://fidoalliance.org/specs/u2f-specs-master/fido-u2f-overview.html) security keys. Using U2F keys makes the credentials phishing-resistant, lowering the risk of malicious access to your email accounts from being used to launch an internal BEC attempt.
+2. Configure email security features such as [SPF, DMARC, and DKIM](https://support.google.com/a/answer/2466580?product_name=UnuFlow&hl=en&visit_id=637919491839685136-2075570827&rd=1&src=supportwidget0&hl=en). These help make it harder to impersonate your domain name in an email.
+3. Configure spam filters to protect your organization from people impersonating employees from other domain names. On Google Workspace/Gmail, this setting is called **Protect against spoofing of employee names**, and you should enable it along with **Protect against inbound emails spoofing your domain**. 
+
+You can find detailed instructions on configuring Google Workspace in our [handbook](https://fleetdm.com/handbook/security#email-authentication).
+
+## Online banking configuration
+
+The best email configuration ever is not enough to protect from BEC and other types of social engineering attacks. Something will always slip by, and we must assume that even well-trained employees will fall for a good enough attempt. 
+
+If you think you can't fall victim to these emails, you probably have not received one that was sufficiently well-targeted and delivered at precisely the right time.
+
+As defenders, we often say that while we must always be perfect, attackers only need to succeed once. As [Joe Slowik](https://www.dragos.com/blog/industry-news/the-myth-of-the-adversary-advantage/) explained in this excellent blog post a couple of years ago, that is seldom true. As defenders, we have the high ground, but we often fail to leverage it.
+
+Since we know the attackers are looking for money, instead of trying to train people to be perfect at spotting scams, which is impossible, let's secure the cash!
+
+### Authentication
+
+If someone can break into your bank accounts directly, they won't need to come up with a convincing email or text message to convince you to move money. It is therefore critical to enable the most advanced security features your bank offers. Unfortunately, in North America, many banks fail to offer any 2FA or only offer SMS-based 2FA, which is the weakest form of 2FA but is still better than nothing.
+
+0. Keep the number of people with online banking access as low as possible.
+1. If available, enable 2FA. Prefer FIDO U2F (very rare right now with banks) where possible and app-based 2FA otherwise. As a last resort, use SMS 2FA.
```

Comment:
```suggestion
1. If available, enable 2FA. FIDO U2F, although very rare right now with banks, is preferable. Otherwise, use app-based 2FA. As a last resort, use SMS 2FA.
```
I rephrased this to remove the sentence fragment, "Prefer."